src/Hosting/Hosting/src/Internal/HostingMetrics.cs

```diff
@@ -13,26 +14,21 @@ internal sealed class HostingMetrics : IDisposable
     public const string MeterName = "Microsoft.AspNetCore.Hosting";
 
     private readonly Meter _meter;
-    private readonly UpDownCounter<long> _currentRequestsCounter;
+    private readonly UpDownCounter<long> _activeRequestsCounter;
     private readonly Histogram<double> _requestDuration;
-    private readonly Counter<long> _unhandledRequestsCounter;
 
     public HostingMetrics(IMeterFactory meterFactory)
     {
         _meter = meterFactory.Create(MeterName);
 
-        _currentRequestsCounter = _meter.CreateUpDownCounter<long>(
-            "http-server-current-requests",
+        _activeRequestsCounter = _meter.CreateUpDownCounter<long>(
+            "http.server.active_requests",
             description: "Number of HTTP requests that are currently active on the server.");
 
         _requestDuration = _meter.CreateHistogram<double>(
-            "http-server-request-duration",
+            "http.server.duration",
```

Comment:
```suggestion
            "http.server.request.duration",
```

We're renaming them in otel - https://github.com/open-telemetry/semantic-conventions/pull/224. Not merged yet, but approved and will be merged soon.